### PR TITLE
Add script_path

### PR DESCRIPTION
**Pull request by omeroguz45**
added scrip_path var with type pathlib.PosixPath that gives the path of the current path script that is being run

### CODE REVIEW

**Review by chamini2 — APPROVED** [review]
Is this testable? If yes, could we add a test?

File: src/fal/run_scripts.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class CurrentTest:
 @dataclass
 class ContextConfig:
     target_path: Path
+    script_path: Path
```
Comment (chamini2) [inline src/fal/run_scripts.py:42]: Curious why we would want script path and not the directory instead? Or the `project_dir` even?

**Review by chamini2 — CHANGES_REQUESTED** [review]

File: src/fal/run_scripts.py (hunk 1 above)
Comment (omeroguz45) [reply src/fal/run_scripts.py:42]: In the data drift example, I wanted to import the anomaly detection functions to use them in the data drift script, and the only way I could do that was to add the dir in which the functions were to the `sys.path`. I brought this up at a standup and we decided that we would add this. [Here's](https://github.com/fal-ai/fal_dbt_examples/blob/283305fac31213ea1cd2dd81b9852a0fd477c9b7/fal_scripts/data_drift.py#L3) where I used it.

**Comment by chamini2** [comment]
Agreed this is no longer necessary since now we offer imports in scripts: https://github.com/fal-ai/fal/pull/190

**Pull request closed without merging**